country_by_country/utils/utils.py

```diff
@@ -49,20 +49,12 @@ def gather_tables(
     assets: dict,
 ) -> dict:
     tables_by_name = {}
-    for algo, algo_assets in assets["text_table_extractors"].items():
-        tables = algo_assets["tables"]
+    for asset in assets["table_extractors"]:
+        tables = asset["tables"]
         if len(tables) == 1:
-            tables_by_name[algo] = tables[0]
+            tables_by_name[asset["type"]] = tables[0]
         if len(tables) > 1:
```

Comment:
```suggestion
        elif len(tables) > 1:
```
Nitpicking, it's fine w/o in this case but in general it's probably better to be explicit since otherwise more than one if statement condition could happen to be true.